tfs/__init__.py

```diff
@@ -1,6 +1,16 @@
 """Exposes TfsDataFrame, read_tfs and write_tfs directly in tfs namespace."""
-from tfs.handler import read_tfs, write_tfs, TfsDataFrame
-__version__ = "1.0.5"
+from tfs.handler import TfsDataFrame, read_tfs, write_tfs
+
+__title__ = "tfs-pandas"
+__description__ = "Read and write tfs files."
+__url__ = "https://github.com/pylhc/tfs"
+__version__ = "2.0.0"
+__author__ = "pylhc"
+__author_email__ = "pylhc@github.com"
+__license__ = "MIT"
+
 # aliases
 read = read_tfs
 write = write_tfs
+
+__all__ = [read, read_tfs, write, write_tfs, __version__]
```

Comment:
Add ```TfsDataFrame```. Should we keep here ```read_tfs``` and ```write_tfs```? or just the short variants? Anything else?

Reply:
As discussed, keeping `read`, `write`, `TfsDataFrame` and `__version__`